includes/class-magic-link.php

```diff
@@ -1153,7 +1197,7 @@ public static function newsletters_email_verification_email( $email, $user, $url
 		$verification_url   = \add_query_arg(
 			[
 				'action'   => self::AUTH_ACTION,
-				'email'    => urlencode( $user->user_email ),
+				'email'    => self::generate_secret( $user ),
```

Comment:
I'm not sure how to trigger `newsletters_email_verification_email` to test in the UI. Would love some direction on this.

Reply:
This is Newspack Newletters' email verification for managing newsletters subscriptions. Under RAS this verification is automatically done when the reader sets a password or connects via magic link/otp.

You can test it by manually removing the newsletter verification in wp shell: `wp user meta delete {user_id} newspack_newsletters_email_verified`. Visit "My Account -> Newsletters" and click to verify.

Reply:
Thanks! Just tested and seems to be working. Also noting I added a quick small commit to fix updating the key here: https://github.com/Automattic/newspack-plugin/pull/3476/commits/6e813075b6c7d6e608fa15f76cab0ce5c85bc539